temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package temporal.api.cloud.namespace.v1;
+
+option go_package = "go.temporal.io/api/cloud/namespace/v1;namespace";
+
+import "google/protobuf/timestamp.proto";
+
+message CertificateFilterSpec {
+    // The common_name in the certificate
+    string common_name = 1;
+    // The organization in the certificate
+    string organization = 2;
+    // The organizational_unit in the certificate
+    string organizational_unit = 3;
+    // The subject_alternative_name in the certificate
+    string subject_alternative_name = 4;
+}
+
+message CodecServerPropertySpec {
+    // Server endpoints
```

Comment:
nit - codec server endpoint

Also, does this take in 1 endpoint or many? the plural endpoints in the comment is ambiguous

Reply:
Its just one, fixed the comments.